django/utils/http.py

```diff
@@ -33,6 +33,12 @@
 RFC3986_GENDELIMS = str(":/?#[]@")
 RFC3986_SUBDELIMS = str("!$&'()*+,;=")
 
+PROTOCOL_TO_PORT = {
+    'http': 80,
+    'https': 443,
+    'ftp': 21,
```

Comment:
I'm wondering if there is some Python builtin with this mapping. I didn't find any at a first glance.